openwisp_radius/tests/test_celery.py

```diff
@@ -0,0 +1,79 @@
+from celery import Celery
+from celery.contrib.testing.worker import start_worker
+from django.contrib.auth import get_user_model
+
+from openwisp_radius import tasks
+
+from ..utils import load_model
+from . import _RADACCT, FileMixin
+from .mixins import BaseTestCase
+
+User = get_user_model()
+RadiusAccounting = load_model('RadiusAccounting')
+RadiusBatch = load_model('RadiusBatch')
+RadiusPostAuth = load_model('RadiusPostAuth')
+
+
+class TestCelery(FileMixin, BaseTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        app = Celery('openwisp2')
+        app.config_from_object('django.conf:settings', namespace='CELERY')
+        app.autodiscover_tasks()
+        cls.celery_worker = start_worker(app)
```

Comment:
are you starting the worker here? We aren't doing this in the other module but I want to understand more about it.
Can you explain a bit more what this test class does? Adding a few comments would be very useful.